src/dlsproto/client/request/internal/GetRange.d

```diff
@@ -465,8 +465,12 @@ private scope class GetRangeHandler
                 if (!(*input).length)
                     enableStomping(*input);
 
-                (*input).length = data.length;
-                (*input)[] = data[];
+                // Note that the appending here is needed, since the node
+                // will send the remaining batch if received the Stop message
+                // during sending the batch.
+                auto old_len = (*input).length;
```

Comment:
Isn't this the same as doing `(*input) ~= data;`?

Reply:
Yeah, good point :man_facepalming: